yarn.lock

```diff
@@ -10813,7 +10838,7 @@ jsii-rosetta@~5.4.10:
     workerpool "^6.5.1"
     yargs "^17.7.2"
 
-jsii@~5.4.0, jsii@~5.4.9:
+jsii@~5.4.0:
   version "5.4.9"
```

Comment:
Should this be `5.4.0` since `5.4.9` was just removed?

Reply:
Huh. Not sure why that didn't update initially. I've forced it now. Ready for another review.